p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -395,5 +395,14 @@ Layout Description
         <sha512Version/>
         <readOnly/>
         <ecc/>
-    </section>    
+    </section>
+    <section>
+        <description>Ultravisor XSCOM White/Blacklist (64K)</description>
+        <eyeCatch>UVBWLIST</eyeCatch>
+        <physicalOffset>0x31F4000</physicalOffset>
+        <physicalRegionSize>0x10000</physicalRegionSize>
+        <side>sideless</side>
+        <sha512Version/>
+        <readOnly/>
+    </section>
```

Comment:
Is there a reason this doesn't have ecc?  The sha hash protects from corruption but it doesn't allow us to fix single bit flips.

Reply:
The decision was to have the same ECC set up on this partition as on UVSIOR, which is no ECC.